### PR TITLE
[ci-skip] Add README.md files

### LINKED ISSUE #2285: feat(corda-os): update platform helm charts with README.md
### Description
As a developer I want to add proper documentation for the helm charts, before they are packaged and published on bevel helm repo

### Acceptance Criteria
1. README.md file added.
2. vaules.yaml file filled with default values.
3. chart.yaml versions updated.

### PR DESCRIPTION
**Pull request by sailajakommineni**
Primary Changes
--------------
1. Add readme files for all charts
2. Update the values.yaml files
3. Update the version in chart.yaml file

Modifications in charts
platforms/r3-corda/charts/doorman-tls
platforms/r3-corda/charts/doorman
platforms/r3-corda/charts/generate-certs
platforms/r3-corda/charts/h2-addUser
platforms/r3-corda/charts/h2-password-change
platforms/r3-corda/charts/h2
platforms/r3-corda/charts/mongodb-tls
platforms/r3-corda/charts/mongodb
platforms/r3-corda/charts/nms-tls
platforms/r3-corda/charts/nms
platforms/r3-corda/charts/node-initial-registration 
platforms/r3-corda/charts/node 
platforms/r3-corda/charts/notary-initial-registration 
platforms/r3-corda/charts/notary platforms/r3-corda/charts/storage

Fixes #2285

### CODE REVIEW

File: platforms/r3-corda/charts/generate-certs/Chart.yaml [hunk 1]
```diff
@@ -6,6 +6,6 @@
 
 apiVersion: v1
 appVersion: "2.0"
-description: A Helm chart for generating the certificates for Corda Opensource
+description: "R3-corda-os: Its generates the certificates for Corda Opensource"
```
Comment (sownak) [inline platforms/r3-corda/charts/generate-certs/Chart.yaml:9]: just ": Generates ..."

File: platforms/r3-corda/charts/h2-addUser/Chart.yaml [hunk 2]
```diff
@@ -6,6 +6,6 @@
 
 apiVersion: v1
 appVersion: "2.0"
-description: A Helm chart for registering the notary with the nms
-name: h2-pass-change
-version: '0.14.0'
+description: "R3-corda-os: Its authenticate with the Vault server for Corda Opensource"
```
Comment (sownak) [inline platforms/r3-corda/charts/h2-addUser/Chart.yaml:9]: This chart is not for authenticating use. If this is never used from the roles or playbook, remove these charts

**Review by sownak — CHANGES_REQUESTED** [review]
Also make the descriptions concise. Like no need to add "on Kubernetes" because helm always deploys on Kubernetes and not anywhere else.

File: platforms/r3-corda/charts/generate-certs/Chart.yaml [hunk 3]
```diff
@@ -6,6 +6,6 @@
 
 apiVersion: v1
 appVersion: "2.0"
-description: A Helm chart for generating the certificates for Corda Opensource
+description: "R3-corda-os: Generates the ca-certificates for doorman using openssl."
```
Comment (sownak) [inline platforms/r3-corda/charts/generate-certs/Chart.yaml:9]: This is not only for doorman

File: platforms/r3-corda/charts/node-initial-registration/Chart.yaml [hunk 4]
```diff
@@ -6,6 +6,6 @@
 
 apiVersion: v1
 appVersion: "2.0"
-description: A Helm chart for Kubernetes
+description: "R3-corda-os: Deploys the job for the initial node registration."
```
Comment (sownak) [inline platforms/r3-corda/charts/node-initial-registration/Chart.yaml:9]: "Job for initial node registration"

File: platforms/r3-corda/charts/notary-initial-registration/Chart.yaml [hunk 5]
```diff
@@ -6,6 +6,6 @@
 
 apiVersion: v1
 appVersion: "2.0"
-description: A Helm chart for Kubernetes
+description: "R3-corda-os: Deploys the job for initial notory node registration."
```
Comment (sownak) [inline platforms/r3-corda/charts/notary-initial-registration/Chart.yaml:9]: "Job for initial notary node registration"

File: platforms/r3-corda/charts/node/Chart.yaml [hunk 6]
```diff
@@ -6,6 +6,6 @@
 
 apiVersion: v1
 appVersion: "2.0"
-description: A Helm chart for Kubernetes
+description: "R3-corda-os: Deploys the job for r3corda node."
```
Comment (sownak) [inline platforms/r3-corda/charts/node/Chart.yaml:9]: no job involved here

File: platforms/r3-corda/charts/notary/Chart.yaml [hunk 7]
```diff
@@ -6,6 +6,6 @@
 
 apiVersion: v1
 appVersion: "2.0"
-description: A Helm chart for Kubernetes
+description: "R3-corda-os: Deploys the r3corda notory."
```
Comment (sownak) [inline platforms/r3-corda/charts/notary/Chart.yaml:9]: notary node

**Review by sownak — CHANGES_REQUESTED** [review]

**Review by sownak — APPROVED** [review]

File: platforms/r3-corda/charts/doorman/Chart.yaml [hunk 8]
```diff
@@ -6,6 +6,6 @@
 
 apiVersion: v1
 appVersion: "2.0"
-description: A Helm chart for Kubernetes
+description: "R3-corda-os: Deploys the doorman service for r3corda node."
```
Comment (suvajit-sarkar) [inline platforms/r3-corda/charts/doorman/Chart.yaml:9]: can you remove the "for r3corda node" part

File: platforms/r3-corda/charts/mongodb/README.md [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+- [Chart Structure](#chart-structure)
+- [Configuration](#configuration)
+- [Deployment](#deployment)
+- [Contributing](#contributing)
+- [License](#license)
+
+<a name = "Mongodb-deployment-helm-chart"></a>
+## Mongodb Deployment Helm Chart
+---
+This [Helm chart](https://github.com/hyperledger/bevel/tree/develop/platforms/r3-corda/charts/mongodb) depolys Mongodb.
```
Comment (suvajit-sarkar) [inline platforms/r3-corda/charts/mongodb/README.md:20]: typo mistake "deploys"

File: platforms/r3-corda/charts/mongodb-tls/README.md [hunk 10]
```diff
@@ -0,0 +1,158 @@
...
+- [Chart Structure](#chart-structure)
+- [Configuration](#configuration)
+- [Deployment](#deployment)
+- [Contributing](#contributing)
+- [License](#license)
+
+<a name = "Mongodb-tls-deployment-helm-chart"></a>
+## Mongodb-tls Deployment Helm Chart
+---
+This [Helm chart](https://github.com/hyperledger/bevel/tree/develop/platforms/r3-corda/charts/mongodb-tls) deploys a MongoDB to enable tls for doorman and networkmap services.
```
Comment (suvajit-sarkar) [inline platforms/r3-corda/charts/mongodb-tls/README.md:20]: Just say deploys MongoDB with tls enabled

File: platforms/r3-corda/charts/nms-tls/README.md [hunk 11]
```diff
@@ -0,0 +1,178 @@
...
+- [Chart Structure](#chart-structure)
+- [Configuration](#configuration)
+- [Deployment](#deployment)
+- [Contributing](#contributing)
+- [License](#license)
+
+<a name = "nms-tls-deployment-helm-chart"></a>
+## nms-tls Deployment Helm Chart
+---
+This [Helm chart](https://github.com/hyperledger/bevel/tree/develop/platforms/r3-corda/charts/nms-tls) deploys the Kubernetes deployment file for a networkmapservice with tls certificate.
```
Comment (suvajit-sarkar) [inline platforms/r3-corda/charts/nms-tls/README.md:20]: change "tls certificate" to "tls enabled"

File: platforms/r3-corda/charts/nms/README.md [hunk 12]
```diff
@@ -0,0 +1,179 @@
...
+- [Chart Structure](#chart-structure)
+- [Configuration](#configuration)
+- [Deployment](#deployment)
+- [Contributing](#contributing)
+- [License](#license)
+
+<a name = "nms-deployment-helm-chart"></a>
+## nms Deployment Helm Chart
+---
+This [Helm chart](https://github.com/hyperledger/bevel/tree/develop/platforms/r3-corda/charts/nms) deploys the Kubernetes deployment file for a networkmapservice.
```
Comment (suvajit-sarkar) [inline platforms/r3-corda/charts/nms/README.md:20]: Please make it consistent, compare with how you have written for other charts. You can remove "the Kubernetes deployment file for a". Check across all charts

**Review by suvajit-sarkar — CHANGES_REQUESTED** [review]
Changes added as comments

**Review by suvajit-sarkar — APPROVED** [review]
LGTM

**Pull request merged**